predicators/utils.py

```diff
@@ -166,6 +166,24 @@ def segment_trajectory_to_atoms_sequence(
     return atoms_seq
 
 
+def order_data(
```

Comment:
can we name this something more specific?

Reply:
actually, now that I see where this is being used, maybe we don't really need a utility function for this and we can just move this code inside the loop? the fact that it's hard to come up with a specific name is good evidence that we probably won't be using this utility elsewhere. and since it's short, might as well just put it where it's used.